### PR TITLE
Chapter 5 Filling in the layout

### PR DESCRIPTION
**Pull request by luongdm-3080**
Chapter 5 Filling in the layout

- [x] Add bootstrap for stylesheets
- [x] Add i18n support (en/vi/jp)

alt

### CODE REVIEW

**Comment by luongdm-3080** [comment]
Ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+@import "color";
+
+/* universal */
+
+body {
+    padding-top: 60px;
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/custom.scss:8]: e check lại indent nhé. tab 2 space

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<footer class="footer">
+    <small>
+        <%= link_to t("rails_tutorial_name"), Settings.links.rails_tutorial_jp %>
+        <%= link_to t(".tutorial_author"), Settings.links.rails_author   %>
+    </small>
+    <nav>
+        <ul>
+            <li><%= link_to t(".about"), about_path %></li>
```
Comment (ngocvt-0484) [inline app/views/layouts/_footer.html.erb:8]: same

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
   </head>
 
   <body>
-    <%= yield %>
+    <%= render "layouts/header" %>
+
+    <div class="container">
+      <%= yield %>
+    </div>
+      <%= render "layouts/footer" %>
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb:15]: sai indent

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (luongdm-3080) [reply app/assets/stylesheets/custom.scss:8]: fixed

File: app/views/layouts/_footer.html.erb (hunk 2 above)
Comment (luongdm-3080) [reply app/views/layouts/_footer.html.erb:8]: fixed

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (luongdm-3080) [reply app/views/layouts/application.html.erb:15]: fixed

**Comment by luongdm-3080** [comment]
Ready

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", root_path, id: "logo" %>
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb:3]: I18n

File: app/views/static_pages/home.html.erb [hunk 5]
```diff
@@ -1,8 +1,13 @@
...
-  <%= link_to t("rails_tutorial_name"), Settings.links.rails_tutorial_org %>
-  <%= t ".second_intro" %>
-</h2>
+  <h2>
+    <%= t ".first_intro" %>
+    <%= link_to t("rails_tutorial_name"), Settings.links.rails_tutorial_org %>
+    <%= t ".second_intro" %>
+  </h2>
+
+<%= link_to t(".sign_up"), signup_path, class: "btn btn-lg btn-primary" %>
```
Comment (huongnt-2545) [inline app/views/static_pages/home.html.erb:11]: sai indent

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**